chat-plugins/mafia-data.js

```diff
@@ -0,0 +1,99 @@
+// Data for the mafia chat plugin.
+
+'use strict';
+
+// This object contains all functions that execute in the callback function of any mafia class. Executed from the context of the executing player.
+// Target is a MafiaPlayer object.
+let MafiaFunctions = {
+	copReport: function (target) {
+		if (target.class.side === 'village') {
+			return 'After investigating ' + target.name + ' you find out they\'re sided with the village.';
```

Comment:
Convention is to use double quotes for human-readable text.
Then you no longer need to escape the single quotes.